diff -ru original_ext_aio/plugins/vmcustom/textinput/textinput.php patched_ext_aio/plugins/vmcustom/textinput/textinput.php
--- original_ext_aio/plugins/vmcustom/textinput/textinput.php	2021-01-29 20:04:46.000000000 +0100
+++ patched_ext_aio/plugins/vmcustom/textinput/textinput.php	2021-11-06 12:26:00.000000000 +0100
@@ -218,7 +218,7 @@
      * @param $name
      * @param $render
      * @return false
-     */
+     *//*
 	function plgVmOnAddToCartFilter (&$product, &$customfield, &$customProductData, &$customFiltered){
 
         if ($customfield->custom_element != $this->_name ) {
@@ -244,7 +244,7 @@
             }
 	        //vmdebug('There is something',$customProductData);
         }
-    }
+    }*/
 
     /**
      * Trigger to manipulate the cart after adding a product.
@@ -264,7 +264,26 @@
 		}
 		$render->html = '';
 	}
+	
+  // this not permit to change value !
+  public function plgVmOnAddToCartFilter($product, $customfield, $customProductData, $customFiltered){
+    if ($customfield->custom_element !==$this->_name) {
+      return ;
+    }
+    if(empty($customProductData[$customfield->virtuemart_custom_id])) {
+      echo json_encode(array('stat' =>1, 'msg' => "Bitte erst die Detail-Angaben ausf&uuml;llen!"));
+      jexit();
+    } else if(empty($customProductData[$customfield->virtuemart_custom_id][$customfield->virtuemart_customfield_id])) {
+      return ;
+    }
+
+    $selected = $customProductData[$customfield->virtuemart_custom_id][$customfield->virtuemart_customfield_id];
+    if( !$selected['comment'] ) {
+      echo json_encode(array('stat' =>1, 'msg' => "Bitte Eingabefeld '$customfield->custom_title' ausf&uuml;llen."));
+      jexit();
+    }
+  }	
 
 }
 
-// No closing tag
\ Kein Zeilenumbruch am Dateiende.
+// No closing tag
