diff -ru original/components/com_virtuemart/sublayouts/addtocartbar.php patched/components/com_virtuemart/sublayouts/addtocartbar.php
--- original/components/com_virtuemart/sublayouts/addtocartbar.php	2021-01-29 20:04:38.000000000 +0100
+++ patched/components/com_virtuemart/sublayouts/addtocartbar.php	2021-11-06 11:56:00.000000000 +0100
@@ -90,7 +90,12 @@
 	// Display the quantity box
 	$stockhandle = VmConfig::get('stockhandle_products', false) && $product->product_stockhandle ? $product->product_stockhandle : VmConfig::get('stockhandle','none');
 	if ($product->show_notify) { ?>
-        <a class="notify <?php echo $vmbtn.' '.$vmbtnsec ?>" href="<?php echo JRoute::_ ('index.php?option=com_virtuemart&view=productdetails&layout=notify&virtuemart_product_id=' . $product->virtuemart_product_id); ?>" ><?php echo vmText::_ ('COM_VIRTUEMART_CART_NOTIFY') ?></a><?php
+<!--        <a class="notify <?php echo $vmbtn.' '.$vmbtnsec ?>" href="<?php echo JRoute::_ ('index.php?option=com_virtuemart&view=productdetails&layout=notify&virtuemart_product_id=' . $product->virtuemart_product_id); ?>" ><?php echo vmText::_ ('COM_VIRTUEMART_CART_NOTIFY') ?></a><?php-->
+        
+    <b>Fahrt ist ausgebucht!</b> Bitte wenden Sie sich an den Organisator.  
+  <?php        
+        
+        
 	} else {
 		$tmpPrice = (float) $product->prices['costPrice'];
 		if (!( VmConfig::get('askprice', true) and empty($tmpPrice) ) ) {
